staging/konvoyconfig/values.yaml

```diff
@@ -0,0 +1,9 @@
+# Default values for konvoyconfig
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+# The cluster hostname is a FQDN which should resolve to the clusters ingress VIP
+clusterHostname: ""
```

Comment:
I believe we will create more properties in the future within this configMap. Let's create them under an object `config`:
```
config:
    clusterHostname:
```